src/openMVG/multiview/solver_triangulate_nviews_LORansac_test.cpp

```diff
@@ -0,0 +1,77 @@
+#include <openMVG/multiview/solver_triangulate_nviews_LORansac.hpp>
+#include <openMVG/robust_estimation/score_evaluator.hpp>
+#include <testing/testing.h>
+
+#include <vector>
+
+using namespace openMVG;
+
+// Test triangulation as algebric problem, it generates some random projection
```

Comment:
typo algebric

Reply:
thx! I forgot to remove these files (moved to multiview/)